client/lib/plans/constants.js

```diff
@@ -124,11 +142,30 @@ export const PLANS_LIST = {
 			FEATURE_BASIC_DESIGN,
 			FEATURE_3GB_STORAGE
 		],
-		getBillingTimeFrame: () => i18n.translate( 'for life' )
+		getSignupFeatures: () => [ // pay attention to ordering, it is used on /plan page
+			FEATURE_COMMUNITY_SUPPORT,
+			FEATURE_WP_SUBDOMAIN_SIGNUP,
+			FEATURE_FREE_THEMES_SIGNUP,
+		],
+		getBlogSignupFeatures: () => [ // pay attention to ordering, it is used on /plan page
+			FEATURE_COMMUNITY_SUPPORT,
+			FEATURE_WP_SUBDOMAIN_SIGNUP,
+			FEATURE_FREE_THEMES_SIGNUP,
+		],
+		getPortfolioSignupFeatures: () => [ // pay attention to ordering, it is used on /plan page
+			FEATURE_COMMUNITY_SUPPORT,
+			FEATURE_WP_SUBDOMAIN_SIGNUP,
+			FEATURE_FREE_THEMES_SIGNUP,
+		],
+		getBillingTimeFrame: () => i18n.translate( 'for life' ),
+		getSignupBillingTimeFrame: () => i18n.translate( ' for life' ),
```

Comment:
Does it need to be translated for A/B test? All other `getSignupBillingTimeFrame()` return non-translated strings.